Common/src/main/java/com/blamejared/crafttweaker/platform/services/IRegistryHelper.java

```diff
@@ -31,14 +38,34 @@
 import net.minecraft.world.level.biome.Biome;
 import net.minecraft.world.level.block.Block;
 import net.minecraft.world.level.material.Fluid;
-import javax.annotation.Nullable;
 
+import javax.annotation.Nullable;
 import java.util.List;
 import java.util.Optional;
 
 public interface IRegistryHelper {
     
-    void initRegistries();
+    default void registerSerializer(MappedRegistry<IIngredientTransformerSerializer<?>> registry, IIngredientTransformerSerializer<?> serializer) {
```

Comment:
Comment here, but for the entire set of changes:
Since we have this as a service, it might be worth to split this into a Forge version and a Fabric version.

Reply:
Forge registries require their entries implement `IForgeRegistryEntry`, which our serializers do not